modules/ding_ting_frontend/ding_ting_frontend.field_group.inc

```diff
@@ -605,9 +506,9 @@ function ding_ting_frontend_field_group_info() {
     'label' => 'Right column meta',
     'weight' => '1',
     'children' => array(
-      0 => 'ting_title',
-      1 => 'ting_type',
-      2 => 'ting_author',
+      0 => 'ting_author',
+      1 => 'ting_title',
+      2 => 'ting_type',
```

Comment:
This seems to reorder elements. Can this be right?

Reply:
I think it is... I believe that the order here is irrelevant, it's just a list of which fields belong in the fieldgroup (which sadly seems to be numerically indexed), I think it still uses field weights to determine rendering order.

Note that it has alphabetized the lists.